website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,317 @@
+
+```go
+...
+var namespaces []string // List of Namespaces
+
+mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+    MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a namespace that 
+is not in this List of `namespaces` then, its controller will not re-trigger its reconcile since the namespaced is NOT 
+part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding namespaces, this is better done via a Predicate. 
+Also note that you may face performance issues when using this with a high number of namespaces.
+
+## Restricting Roles and permissions
+
+The `operator-scope` defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. 
+In this scenario, it would also be recommended to change the default 
+[Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] to be restrictive accordingly.
+
+In the projects, the permissions are specified in the directory `config/rbac/`. Note that there you will find `role.yaml` 
+and `role_binding.yaml` files which are to allow permissions across the cluster by default for the operator. 
+
+**NOTE** The files `auth_proxy_client_clusterrole.yaml`, `auth_proxy_role.yaml`, `auth_proxy_role_binding.yaml` are also 
+defined to allow permissions across the cluster as well and are used by the operator. More info see: 
+[kube-rbac-proxy][kube-rbac-proxy]. Also, it is important to highlight that it is an alpha feature that is not officially 
+supported and can be disabled by commenting the lines in the config `config/rbac/kustomization.yaml` and in the 
+`config/default/kustomization.yaml`. See:
+
+-  In `config/rbac/kustomization.yaml`:
+```yaml
+...
+# Comment the following 4 lines if you want to disable
+# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
+- auth_proxy_service.yaml
+- auth_proxy_role.yaml
+- auth_proxy_role_binding.yaml
+- auth_proxy_client_clusterrole.yaml
+```
+
+- In `config/default/kustomization.yaml`:
+```yaml
+...
+patchesStrategicMerge:
+  # Protect the /metrics endpoint by putting it behind auth.
+  # If you want your controller-manager to expose the /metrics
+  # endpoint w/o any authn/z, please comment the following line.
+- manager_auth_proxy_patch.yaml
+...
+```
+
+And then, see that for each API (CRD) scaffolded a specific role file with permissions to `edit` or `view` the Kind 
+resource in the cluster is created too. It is not used by the project, but are helpers to allow you to perform specific 
+configurations. For example, after creating a new API(CRD) with the Memcached Kind, we will able to see the roles 
+`memcached_editor_role.ymal` and `memcached_viewer_role.yaml` created there.
```

Comment:
@camilamacedo86 While I agree it is important to point all this out, this is probably not the right section. 
This part is derailing the flow of the doc where I only really care about the steps for changing `role.yaml` and `rolebinding.yaml`.

Let's keep this section strictly for the changes needed to change the operator's scope.
And move this to a caveats section at the end of this doc that points out what all the other rbac manifests are and why we might want to disable kube-rbac proxy.

With this, you can probably combine this section `Restricting Roles and permissions` and  `Changing the permissions` into one which is more concise and directly related to the permissions for the operator's scope.

@joelanford I know you had opinions on whether we should be documenting how to disable kube-rbac proxy downstream at all.

Reply:
Hi @hasbro17, 

Note that we need to make these notes here since: 
- Users might need to disable or change the proxy roles to achieve the goal. So, a small note as this without too many details allow they know where and how.  
- When users will be following these steps will check all these permissions files, so common questions will be raised such as: Has been these permissions also been applied and etc?. Then, IHMO it is important to keep a NOTE here.

Reply:
@camilamacedo86 This note is hardly small. It's a complete section that diverts me into reading about kube-rbac proxy and how to disable it.

My point is that it's not necessary to modify the other ClusterRole/ClusterRolebinding manifests in `config/rbac` besides `role.yaml` and `rolebinding.yaml` just to change the permissions for a namespace scoped operator.
Please clarify if it is necessary.

So I still say we have to move this section to the end of this doc into a caveats section.

Reply:
Addressed as suggested.